### ISSUE TITLE
Using JSCS as JSX Beautifier

### ISSUE THREAD

**Issue by expelledboy** [Glavin001/atom-beautify#929]
Not sure if this is already supported, but I have searched through the issues.

In the configuration you can use "JSCS Fixer" as the javascript beautifier. JSCS Fixer supports JSX as well.

Would it be possible to add the option to use "JSCS Fixer" for JSX in the configuration editor?

**Comment by Glavin001** [comment]
Good idea. The language `JSX` can definitely be added to the `JSCS Fixer` beautifier.

A Pull Request adding `JSX: true` to the list of supported language options found at https://github.com/Glavin001/atom-beautify/blob/master/src/beautifiers/jscs.coffee#L12 would do it.

**Comment by expelledboy** [comment]
I wasnt sure how to test [my fork](https://github.com/expelledboy/atom-beautify), so replicated the changes on the copy installed by atom. The option doesnt seem to work?

**Comment by prettydiff** [comment]
@expelledboy Just make the appropriate changes to the Atom-Beautify files and then restart Atom.  Atom has to be restarted in order to reinitialize the application from changes in the code.  Once it works on your local push the changes from your local to your fork.  Then open a pull request, at which point the Travis.CI unit tests should fire.

**Comment by expelledboy** [comment]
Thanks. I assumed a restart was required, but I noticed that a second background instance existed, so after killing that and restarting it still does not seem to work?

```
10:08+ ~/.atom/packages
@ anthony-mac> grep -A 3 options: ~/.atom/packages/atom-beautify/src/beautifiers/jscs.coffee
  options: {
    JavaScript: false
    JSX: true
  }

10:08+ ~/.atom/packages
@ anthony-mac> ps -ef | grep atom
  501 46878 32036   0 10:08AM ttys004    0:00.00 grep --color=auto atom

10:08+ ~/.atom/packages
@ anthony-mac> atom

10:08+ ~/.atom/packages
@ anthony-mac> ps -ef | grep atom | wc -l
       6
```

When going to setting only `Pretty Diff` is available. Is there perhaps a cache or a temp file that needs to be reset? Perhaps atom caches the setting options?
